theme/views/@layout.latte

```diff
@@ -1,27 +1,25 @@
 <!DOCTYPE html>
 
-{default $description = ''}
-{default $keywords = ''}
-{default $activeLang = 'cs'}
+{default $activeLang = explode('_', get_locale())[0]}
 
-<html>
+<html lang="{$activeLang}">
 <head>
 	<meta charset="utf-8">
 	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 	<meta name="viewport" content="width=device-width,initial-scale=1">
 
-	<title>{site_title()}</title>
+	{site_title()}
```

Comment:
AFAIK it's much easier to create Nette Html element with wrapping `<title />` tag.
https://github.com/manGoweb/MangoPress/pull/64/files/a62fc3d84fa316ce8c5bdd4999a281703e3ae6cf#diff-6ecff5c8ded1ef98dec31e6b03d293d1R2